packages/office-ui-fabric-react/src/components/ContextualMenu/examples/ContextualMenu.CustomMenuList.Example.tsx

```diff
@@ -2,11 +2,78 @@ import * as React from 'react';
 import { DefaultButton } from 'office-ui-fabric-react/lib/Button';
 import { SearchBox } from 'office-ui-fabric-react/lib/SearchBox';
 import { Icon } from 'office-ui-fabric-react/lib/Icon';
-import './ContextualMenuExample.scss';
 import { IContextualMenuListProps, IContextualMenuItem } from 'office-ui-fabric-react/lib/ContextualMenu';
 import { IRenderFunction } from 'office-ui-fabric-react/lib/Utilities';
 
-const ITEMS: IContextualMenuItem[] = [
+export const ContextualMenuWithCustomMenuListExample: React.FunctionComponent = () => {
+  const [items, setItems] = React.useState(menuItems);
+
+  const onAbort = () => {
+    setItems(menuItems);
+  };
+
+  const onChange = (ev: React.ChangeEvent<HTMLInputElement>, newValue: string) => {
+    const filteredItems = menuItems.filter(item => item.text && item.text.toLowerCase().indexOf(newValue.toLowerCase()) !== -1);
+
+    if (!filteredItems || !filteredItems.length) {
+      filteredItems.push({
+        key: 'no_results',
+        onRender: (item, dismissMenu) => (
+          <div
+            key="no_results"
+            style={{
+              width: '100%',
+              height: '100px',
+              display: 'flex',
+              alignItems: 'center',
+              justifyContent: 'center'
+            }}
+          >
+            <Icon iconName="SearchIssue" title="No actions found" />
+            <span>No actions found</span>
+          </div>
+        )
+      });
+    }
+
+    setItems(filteredItems);
+  };
+
+  const renderMenuList = (menuListProps: IContextualMenuListProps, defaultRender: IRenderFunction<IContextualMenuListProps>) => {
+    return (
+      <div>
+        <div style={{ borderBottom: '1px solid #ccc' }}>
```

Comment:
Again being really pedantic, this `style` and the `styles` below should be in consts at the top or bottom of the file.